src/core/metrics.cc

```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2018-2021, NVIDIA CORPORATION & AFFILIATES. All rights
-// reserved.
+// Copyright (c) 2018-21, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Just remove the '(c)' after Copyright since it is causing the copyright to be split into 2 lines because of the formatter.

Reply:
Fixed this in other places too.